### ISSUE TITLE
getDatabases() should always return a data.frame

### ISSUE THREAD

**Issue by akbertram** [bedatadriven/activityinfo-R#83]
The list of databases can be perfectly represented as a data.frame. There is no reason to return a list. 

@Ryo-N7 will this break anything for you?

**Comment by Ryo-N7** [comment]
no, this shouldn't break anything for me!

**Comment by nickdickinson** [comment]
We could leave the asDf argument but change the default to TRUE so if it is breaking any existing script, it is easy to fix with asDf = FALSE.

**Comment by Ryo-N7** [comment]
i can add that to both `getDatabases()` and `getDatabaseUsers()`

**Comment by Ryo-N7** [comment]
yup done, don't have time to create tests today so i'll try to do that tomorrow
